src/components/ForecastInfo/ForecastInfo.jsx

```diff
@@ -0,0 +1,47 @@
+import PropTypes from 'prop-types';
+import { SearchBar } from 'components/SearchBar/SearchBar';
+import s from 'components/ForecastInfo/ForecastInfo.module.scss';
+
+export const ForecastInfo = ({ Forecastweather, onChangeSubmit }) => {
+  return (
+    <section className={s.section}>
+      <SearchBar onSubmit={onChangeSubmit} />
+      <h1 className={s.forecastTitle}>{Forecastweather.city.name}</h1>
+
+      <div className={s.forecastInfoThumb}>
+        <img
+          className={s.icon}
+          src={`http://openweathermap.org/img/wn/${Forecastweather.list[0].weather[0].icon}@2x.png`}
```

Comment:
what should happens if you don't have list or 0 element in list or you don't have weather or 0 element or icon?

Reply:
Need default Icon. Fix